README.md

```diff
@@ -89,7 +89,7 @@ Some of the most common built-in matchers are
 - `s.object(fields)`
 - `s.objectWithOnly(fields)`
 - `s.regex(reg)`
-- `s.string()`
+- `s.string(min, max)`
```

Comment:
Sorry, I meant I like `s.string({min, max})`... and if we choose that let's update the other ones that take an object param as well.